crates/bevy_render/src/shader/shader_reflect.rs

```diff
@@ -65,15 +66,14 @@ impl ShaderLayout {
                                     .name
                                     .splitn(3, '_')
                                     .collect::<Vec<&str>>();
+
                                 if parts.len() == 3 {
                                     if parts[0] == "I" {
                                         instance = true;
-                                        parts[1].to_string()
-                                    } else {
-                                        parts[0].to_string()
                                     }
+                                    vertex_attribute_descriptor.name.to_string()
                                 } else if parts.len() == 2 {
-                                    parts[0].to_string()
+                                    vertex_attribute_descriptor.name.to_string()
                                 } else {
                                     panic!("Vertex attributes must follow the form BUFFERNAME_PROPERTYNAME. For example: Vertex_Position");
```

Comment:
This convention probably need to be changed as now everything is just a separate buffer